### ISSUE TITLE
MMS image wouldn't send until after sending text message

### ISSUE THREAD

**Issue by Tursko** [signalapp/Signal-Android#12175]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
I created a new MMS thread and tried to send an image and it kept failing. Afterwards I deleted the thread, created a new one, send a text message, and then sent an image and it was successful.

### Steps to reproduce
- Create MMS thread.
- Send image as first message
- Failed to send

**Actual result:** Failed to send
**Expected result:** Should send successfully

### Device info

**Device:** Manufacturer Model XVI
**Android version:** 12
**Signal version:** 5.34.10

### Link to debug log

https://debuglogs.org/android/5.34.10/8fadf4c36a708d0b364262f3be6637330f9d602686d62f4b8e5815a30a420992

**Comment by Tursko** [comment]
Let me better explain what I think happened. 

1. I had an issue via #12087 which caused me to delete an MMS thread with two contacts. This thread had a custom name and a custom picture. 
2. After deleting it, I had one of my contacts send in that thread again. When I got the messages the new thread I noticed that it somehow still had a reference to the custom chat image I set. So somehow that didn't get cleaned up on delete?
3. Then I tried to send an image in that chat, which failed.
4. I deleted the chat and recreated it. This time it was fresh with no custom image.
5. No issues sending after.
